.github/workflows/deploy.yml

```diff
@@ -0,0 +1,30 @@
+name: CD
+
+on:
+  push:
+    tags:
+      - 'v*.*.*' # Enforce Semantic Versioning
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+ 
+    steps: 
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Setup CD Environment
+      uses: yetanalytics/actions/setup-env@v0.0.2
+
+    - name: Extract version
+      id: version
+      run: echo ::set-output name=VERSION::${GITHUB_REF#refs\/tags\/v}
+
+    - name: Build and deploy to Clojars
+      uses: yetanalytics/actions/deploy-clojars@v0.0.2
+      with:
+          artifact-id: 'xapi-schema'
+          resource-dirs: '[]'
```

Comment:
I *think* this might have resource dirs used by downstream cljs apps. Looking into it.

Reply:
Normally I would've let it default to `'["resources"]'` but since you mentioned it can't be used with mvn anyways it's probably less confusing to keep it as `[]`.